tests/unit/test_token.py

```diff
@@ -104,6 +151,22 @@ def test_token_login(self):
             token_controller.load_config = (lambda: self.ss_config)
             token_controller.login()
 
+    def test_token_init_keys(self):
```

Comment:
Here you only test the happy path, please also add tests for failures

Reply:
There was one, added another for where no CA-s are available at all.